src/src.pro

```diff
@@ -129,6 +129,9 @@ macx {
     LIBS += -L/usr/local/lib
     INCLUDEPATH += /usr/local/include
 }
+windows {
+    DEFINES *= Q_COMPILER_INITIALIZER_LISTS
```

Comment:
Could you please show the evidence for adding this config? Btw, please add an empty line before this code block.